libeos-payg/efi.c

```diff
@@ -579,6 +580,34 @@ eospayg_efi_setupmode_active (void)
   return !!content[0];
 }
 
+/* eospayg_efi_secureboot_setup_active:
+ *
+ * Check if the system was booted with SecureBoot Setup config
+ *
+ * Some system does not have EFI variable SecureBootSetup. So, this will return
+ * %FALSE, too. The caller has to check the @has, if it does not have EFI
+ * variable SecureBootSetup.
```

Comment:
Personally I don't particularly like this API. I would prefer it to return a value from a 3-state enum. But if you prefer this API, please rename the out parameter to `found`.